types/lolex/lolex-tests.ts

```diff
@@ -79,9 +109,19 @@ nodeClock.setSystemTime(7);
 nodeClock.setSystemTime(new Date());
 
 nodeClock.nextTick(() => undefined);
+nodeClock.queueMicrotask(() => {});
+
+const browserTimersCount: number = browserClock.countTimers();
+const nodeTimersCount: number = nodeClock.countTimers();
+
+let [secs, nanos] = nodeClock.hrtime([0, 0]);
+[secs, nanos] = nodeClock.hrtime();
+
+// shows that typescript successfully infer the return values as numbers.
+secs.toFixed();
+nanos.toExponential();
 
-browserClock.uninstall();
-nodeClock.uninstall();
```

Comment:
Are the browser and node clocks no longer able to to `uninstall();`?

Reply:
browser clocks are able to uninstall.

Reply:
@JoshuaKGoldberg in Lolex source, the method `uninstall()` is only added to clocks that created with `install()`, but clocks that were created with `createClock()` have no `uninstall()`. That's why I removed the uninstallation of `browserClock` and `nodeClock` and added `installedClock` (line 31) and called `uninstall()` on it (line 124).